src/ucs/datastruct/linear_func.h

```diff
@@ -203,8 +203,8 @@ static inline int
 ucs_linear_func_is_equal(ucs_linear_func_t func1, ucs_linear_func_t func2,
                          double epsilon)
 {
-    return (fabs(func1.m - func2.m) < epsilon) &&
-           (fabs(func1.c - func2.c) < epsilon);
+    return (fabs(func1.m - func2.m) <= epsilon) &&
+           (fabs(func1.c - func2.c) <= epsilon);
```

Comment:
This change is made since it is possible to compare function with 0 epsilon (when we compare two functions which were set by the completely equal way), but the old implementation always returns false in that case.